optuna_dashboard/_config.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import sys
+from typing import Any
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    try:
+        import tomli as tomllib
+    except ImportError:
+        raise ImportError("tomli is required for Python < 3.11. Install with: pip install tomli")
```

Comment:
Can we simply import tomli since it is specified as the mandatory dependency in pyproject.toml?

```suggestion
    import tomli as tomllib
```